packages/forma-36-website/src/content/migration-v3-to-v4.mdx

```diff
@@ -28,6 +28,8 @@ To migrate your project easily, we provided you with a codemod for most of the c
 
 - **Button** API improvements: Some properties of the Button components were migrated or changed. For more information, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-button-components).
 
+- **Card** Some properties of the Card components were migrated or changed. For more information, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-card-components).
```

Comment:
Maybe it's worth mentioning that the guide contains how to migrate `Card`, `EntryCard` and `AssetCard` ?